### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chintanp**
Reformatted the README to have consistent headers.

### CODE REVIEW

**Comment by chintanp** [comment]
@SyXuan can you please add the table of contents to the README like Zander did with BattETL?

**Comment by SyXuan** [comment]
Added. Please review it.

**Review by SyXuan — APPROVED** [review]
LGTM! Add the table of contents

File: README.md [hunk 1]
```diff
@@ -103,17 +121,14 @@ In order to use Docker Compose, you will need to have the following software ins
 
 4. Wait for Docker Compose to complete, and the `BattDB` database has been successfully deployed. The data for the database will be stored in the directory `assets/Docker/data`.
 
-## Migrate Database
+### Using migration Scripts on your server
```
Comment (chintanp) [inline README.md:124]: @SyXuan I am thinking of this as a deployment option too. So this should be Indented further and be the third deployment option. Thanks

Comment (SyXuan) [reply README.md:124]: Thanks for pointing this out!

**Pull request merged**